hls4ml/converters/keras_to_hls.py

```diff
@@ -297,26 +297,18 @@ def parse_keras_model(model_arch, reader):
         layer_list.append(layer)
         if 'activation' in layer and layer['class_name'] not in activation_layers + recurrent_layers:  # + qkeras_layers:
             act_layer = {}
+            act_details = layer['activation']
             # Workaround for QKeras activations passed as an argument
-            if isinstance(layer['activation'], dict):
-                act_details = layer['activation']
+            if isinstance(act_details, dict):
```

Comment:
The logic here is unchanged from before, but is it really valid to say that if the activation is a dictionary, then it is QActivation? Yes, it seems to be used there more often, but can't I pass a leaky relu with a parameter somehow as a dictionary?

Reply:
I was wondering if it was possible to explicitly pass the activation functions (and not text), and if that made a dictionary in this case. But this logic is unchanged from before in this PR. I can try to briefly test it.

Reply:
Actually, it does seem to need to be text:  https://github.com/keras-team/tf-keras/blob/v2.17.0/tf_keras/layers/core/dense.py#L125